design-system-docs/frontend/styles/_footer.scss

```diff
@@ -0,0 +1,37 @@
+    width: 99px;
+    height: 107px;
+    margin-top: 0;
+
+    &:focus {
```

Comment:
This is nitpicky so feel free to ignore - should we also use a different colour border bottom on focus (currently black)

Reply:
Probably yep. What I'll do is come back to the footer logo in a separate PR as I think there's some useful questions there. This code is a bit gross but mirrors what intranet/advisernet do now so we should probably try and fix it up in a more general way.